interface/forms/procedure_order/common.php

```diff
@@ -0,0 +1,1287 @@
+                                                value="Procedure">
+                                        <?php else : ?>
+                                            <input type='hidden' name='form_proc_order_title[<?php echo $i; ?>]'
+                                                value='<?php echo attr($oprow['procedure_order_title']) ?>'>
+                                        <?php endif; ?>
+                                        <input type='text' name='form_proc_type_desc[<?php echo $i; ?>]'
+                                            value='<?php echo attr($oprow['procedure_name']) ?>'
+                                            onclick="sel_proc_type(<?php echo $i; ?>)"
+                                            onfocus='this.blur()'
+                                            title='<?php echo xla('Click to select the desired procedure'); ?>'
+                                            placeholder='<?php echo xla('Click to select the desired procedure'); ?>'
+                                            class='form-control c-hand' readonly />
+                                        <input type='hidden' name='form_proc_type[<?php echo $i; ?>]' value='<?php echo $ptid ?>' />
+                                    </td>
+                                    <td class='diagnosis-div input-group'>
+                                        <div class='input-group-prepend'>
+                                            <span class='btn btn-secondary input-group-text'>
+                                                <i onclick='current_diagnoses(this)' class='fa fa-search fa-lg' title='<?php echo xla('Click to search past and current diagnoses history'); ?>'></i>
+                                            </span>
+                                        </div>
+                                        <input class='form-control c-hand' type='text'
+                                            name='form_proc_type_diag[<?php echo $i; ?>]'
+                                            value='<?php echo attr($oprow['diagnoses']) ?>'
+                                            onclick='sel_related(this.name)'
+                                            title='<?php echo xla('Click to add diagnosis for this test'); ?>'
+                                            onfocus='this.blur()' readonly />
+                                    </td>
+                                    <td>
+                                        <!-- MSIE innerHTML property for a TABLE element is read-only, so using a DIV here. -->
+                                        <div class="table-responsive" id='qoetable[<?php echo attr($i); ?>]'>
+                                            <?php
+                                            $qoe_init_javascript = '';
+                                            echo generate_qoe_html($ptid, $formid, ($oprow['procedure_order_seq'] ?? null), $i);
+                                            if ($qoe_init_javascript) {
```

Comment:
what does this value get populated with? (also noted it used above)
(just trying to figure out if it need js escaping)